### PR TITLE
pgl.yoyo.org now supports https

### LINKED ISSUE #718: Update pgl.yoyo.org to use HTTPS
Please update the URL in Adaway and in the Wiki to use HTTPS.
https://pgl.yoyo.org/as/index.php#https

### PR DESCRIPTION
**Pull request by edent**
Fixes #718

### CODE REVIEW

**Comment by dschuermann** [comment]
I would propose increasing the DB Version and then replacing the entry in update. Can you implementiert that?

**Comment by edent** [comment]
Have I done that correctly?

**Comment by dschuermann** [comment]
You still need to increase the Version counter

File: AdAway/src/main/java/org/adaway/provider/AdAwayDatabase.java [hunk 1]
```diff
@@ -147,7 +147,7 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
             // new hosts source
             db.execSQL("INSERT INTO "
                     + Tables.HOSTS_SOURCES
-                    + " (url, last_modified_local, last_modified_online, enabled) VALUES (\"http://pgl.yoyo.org/adservers/serverlist.php?hostformat=hosts&showintro=0&mimetype=plaintext\", 0, 0, 1)");
+                    + " (url, last_modified_local, last_modified_online, enabled) VALUES (\"https://pgl.yoyo.org/adservers/serverlist.php?hostformat=hosts&showintro=0&mimetype=plaintext\", 0, 0, 1)");
```
Comment (dschuermann) [inline AdAway/src/main/java/org/adaway/provider/AdAwayDatabase.java]: this is just an old version upgrade, no need to change this

**Pull request merged**